system/HTTP/CURLRequest.php

```diff
@@ -616,6 +616,11 @@ protected function setCURLOptions(array $curlOptions = [], array $config = [])
             }
         }
 
+        // DNS Cache Timeout
+        if (! empty($config['dns_cache_timeout']) && is_numeric($config['dns_cache_timeout'])) {
```

Comment:
Can you refrain from using `empty()`? The baseline is supposed to shrink unless refactoring is tedious.

I think this can be isset then is_numeric then !== 0.